tests/Pagerfanta/Tests/View/ViewTestCase.php

```diff
@@ -61,6 +61,13 @@ protected function renderView($options)
         return $this->view->render($this->pagerfanta, $routeGenerator, $options);
     }
 
+    protected function renderViewNeedsEscaping($options)
+    {
+        $routeGenerator = function ($page) { return '|&'.$page.'|'; };
+
+        return $this->view->render($this->pagerfanta, $routeGenerator, $options);
```

Comment:
This could be done by composing the `renderView` method instead of duplicating. :)